packages/base-map/__mocks__/SelectVehicles.js

```diff
@@ -27,7 +29,24 @@ class SelectVehicles extends MapLayer {
 
   farZoom = 10;
 
-  componentDidMount() {}
+  onOverlayAdded() {
+    console.log("SelectVehicles => onOverlayAdded");
```

Comment:
Could we use storybook actions for these so they can be seen without opening the debugger?